roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -96,6 +96,24 @@ kubernetesMasterConfig:
   - v1beta3
   - v1
 {% endif %}
+{% if openshift.master.builddefaults_http_proxy or openshift.master.builddefaults_https_proxy
+   and openshift.common.version_gte_3_2_or_1_2 %}
+  admissionConfig:
+    pluginConfig:
+      BuildDefaults:
+        configuration:
+          apiVersion: v1
+          kind: BuildDefaultsConfig
+          gitHTTPProxy: {{ openshift.master.builddefaults_git_http_proxy }}
+          gitHTTPSproxy: {{ openshift.master.builddefaults_git_https_proxy }}
+          env:
+          - name: HTTP_PROXY
+            value: '{{ openshift.master.builddefaults_http_proxy }}'
+          - name: HTTPS_PROXY
+            value: '{{ openshift.master.builddefaults_https_proxy }}'
+          - name: NO_PROXY
+            value: '{{ openshift.master.builddefaults_no_proxy }}'
```

Comment:
Not related to this PR, but it seems a bit redundant to have to set HTTP_PROXY, HTTPS_PROXY and NO_PROXY for the BuildDefaults here...  it would be nice if the values where injected by the Build Controller, since they are already present in the controllers env.